workitem/workitemtype_repository.go

```diff
@@ -154,6 +156,18 @@ func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UU
 		return nil, errors.NewInternalError(err.Error())
 	}
 
+	// create relationship between workitemtype and category
+	if categoryID != uuid.Nil {
```

Comment:
@kwk `description` is of type `string`. So, it makes sense to use `nil` for it.  `uuid.Nil` is more specific to `UUID` type. Instead of generalizing empty values for every type is it not better to use specific empty values? 

Even in the case of workitem `assignees` which is stored in a slice. Isnt it better to pass empty slice `[]` instead of `None`?

What do you think?